client.go

```diff
@@ -78,6 +78,24 @@ var DefaultErrorUnmarshaler = ErrorUnmarshaler(new(RemoteError))
 // Any error that c.UnmarshalError or c.Doer returns will not
 // have its cause masked.
 func (c *Client) Call(params, resp interface{}) error {
+	return c.CallURL(c.BaseURL, params, resp)
+}
+
+// CallURL invokes the endpoint implied by the given url and params,
```

Comment:
I think I'd make the overlap with Call more explicit:

    // CallURL is like Call except that the given URL is used
    // instead of c.BaseURL.

?